### PR TITLE
Enterprise docs redesign

### PR DESCRIPTION
**Pull request by wladimiiir**
- global header & styling is not changing
- left-hand collapsed Table of Contents for entire /enterprise docs
- ‘Hero image’ for Enterprise
- introduction text body "Welcome to Kong Enterprise Docs...."
- new tiles (with a new tile design, icons) emphasizing "Foundations" and "Solution guides"
- centered search bar
- "What's New" callout text UI element at top
- "Other Resources" section at the bottom with icons linking out to Community, Support, Kong University
- feedback widget “Was this page useful” on all pages
- dynamic left-hand table of contents highlighting the section and page currently navigated to
- dynamic right-hand table of contents with navigation corresponding to the headers on a particular page
- support for ‘tabs’ of content within an individual topic
- enterprise installation options redesign to match community version

### CODE REVIEW

File: app/_layouts/docs-ee.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    </div>
+  </header>
+
+  <div class="container">
+    {% include docs-sidebar.html %}
+
+    {% if page.toc == true or page.toc != false and filename != "index.md" and is_homepage == false %}
+    <aside class="docs-toc">
+      <div class="docs-toc-title">On this page</div>
+      {% include toc.html html=content anchor_class="scroll-to" h_min=2 h_max=5 %}
```
Comment (lena-larionova) [inline app/_layouts/docs-ee.html]:
```suggestion
      {% include toc.html html=content anchor_class="scroll-to" h_min=2 h_max=3 %}
```
To clear the noise from lower headers

Comment (lena-larionova) [inline app/_layouts/docs-ee.html]: Can you also slightly indent the h3 (and lower levels, if we decide to keep them)?

Comment (wladimiiir) [reply app/_layouts/docs-ee.html]: I have pushed the change.

**Comment by lena-larionova** [comment]
@wladimiiir I've been testing out the navtabs, and they're not displaying exactly right. Looks like when the page first loads, the tabs all load one after the other vertically:

Screen Shot 2020-03-30 at 9 13 29 AM

And when you click the tabs and click back on the first one, it resolves and displays correctly:
Screen Shot 2020-03-30 at 9 13 38 AM

**Comment by wladimiiir** [comment]
@lena-larionova I have fixed the issue with navtabs.

**Comment by lena-larionova** [comment]
@hutchic PTAL

**Comment by lena-larionova** [comment]
Closing, as this is a duplicate of changes made in https://github.com/Kong/docs.konghq.com-private/pull/394.

**Pull request closed without merging**